android-app-with-rn/android/app/src/main/java/com/example/hybridapp/MyReactApplication.kt

```diff
@@ -25,22 +23,21 @@ class MyReactApplication : Application(), ReactApplication {
         super.onCreate()
 
         val configuration = Configuration.Builder(
-            logsEnabled = true,
-            tracesEnabled = true,
-            crashReportsEnabled = true,
-            rumEnabled = true
+            clientToken = BuildConfig.CLIENT_TOKEN,
+            env = BuildConfig.ENVIRONMENT,
+            variant = "prod"
+        ).build()
+        Datadog.initialize(this, configuration, TrackingConsent.GRANTED)
+        Datadog.setVerbosity(Log.VERBOSE)
```

Comment:
by changing the order of lines we can get logs during initialization

```suggestion
        Datadog.setVerbosity(Log.VERBOSE)
        Datadog.initialize(this, configuration, TrackingConsent.GRANTED)
```